### PR TITLE
Fixes for Tizen GCC

### PR DESCRIPTION
**Pull request by georged-youi**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -112,6 +112,9 @@ check_function_exists (sigaltstack HAVE_SIGALSTACK)
 if(NOT TIZEN_NACL)
   # NaCl has sigaction but does not have all the types that are being used in the code which uses HAVE_SIGACTION.
   check_function_exists (sigaction HAVE_SIGACTION)
+else()
+  set(HAVE_SYS_SYSCALL_H OFF CACHE INTERNAL OFF FORCE)
```
Comment (Halmo) [inline CMakeLists.txt:116]: Now that we have an else maybe we should swap the condition

**Review by Halmo — APPROVED** [review]
Looks fine, what's up with the commit author?

**Comment by georged-youi** [comment]
> Looks fine, what's up with the commit author?

I'm not sure how that has changed.

**Comment by georged-youi** [comment]
the commit author should be fine now

**Pull request merged**